### PR TITLE
Feat/related articles

### PR DESCRIPTION
**Pull request by vyarovoy-sc**
### Checklist for this pull request

- [x] PR is scoped to one task
- [x] Tests included

### Detailed description

Added new **getRelation** entity to get related articles. New entity uses and has the same functionality as getCollection, but turns to another endpoint http://{_api_server_}/{_path_to_collection_endpoint_}/{**_article_id_**}/articles

### CODE REVIEW

File: src/provider/handler/creators/feedCrator.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// @flow
```
Comment (giorgiomazzei79) [inline src/provider/handler/creators/feedCrator.js]: The name of this file should be feedCreator.js

File: src/provider/handler/getCollection.js [hunk 2]
```diff
@@ -1,20 +1,17 @@
...
-  title: params && params.collectionTitle ? params.collectionTitle : "N/A",
-  entry: data.map(mapCollection)
-});
+export const getCollection = (params: ReqCollectionParameters): Promise<Feed | void> => {
+  const url = urlMaker(params);
 
-export const getCollection = (params: ReqParameters): Promise<Feed | void> =>
-  axios
-    .get(urlMaker(params))
+  return axios
```
Comment (giorgiomazzei79) [inline src/provider/handler/getCollection.js]: Please return the whole axios without using brackets{}.
Like .get(urlMaker(params))

File: src/provider/handler/getRelation.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+// @flow
+import type { Feed, ReqRelationParameters, ResItem } from "promiboom";
+import axios from "axios";
+import { urlMaker } from "../../helpers/urlMaker";
+import { validate } from "../../helpers/validate"
+import { feedCreator } from "./creators/feedCrator";
+
+export const getRelation = (params: ReqRelationParameters): Promise<Feed | void> => {
```
Comment (giorgiomazzei79) [inline src/provider/handler/getRelation.js]: Please, keep the structure similar to getCollection.
I'd like all errors to be caught by the axios chain.
In case use a different ring of the chain for the check of the article.
And throw an error in that case (in the chain).

File: plugin-manifest.json [hunk 4]
```diff
@@ -5,16 +5,16 @@
     "https://github.com/applicaster-plugins/zapp-pipes-provider-promiboom"
   ],
   "platform": null,
-  "manifest_version": "1.1.1",
-  "author_email": "g.mazzei@applicaster.com",
-  "author_name": "Giorgio Mazzei",
+  "manifest_version": "1.1.2",
+  "author_email": "yarovoy@scand.com",
```
Comment (giorgiomazzei79) [inline plugin-manifest.json]: In this file please add the other data type

File: scripts/develop.sh [hunk 5]
```diff
@@ -6,8 +6,8 @@ node_modules/.bin/eslint src/**
 
 if [ $? -eq 0 ]; then
   # if flow check passed run node
-  node_modules/.bin/babel-node $1
+  node_modules/.bin/babel-node $@
```
Comment (giorgiomazzei79) [inline scripts/develop.sh]: why this change?

Comment (vyarovoy-sc) [reply scripts/develop.sh]: Without this changes you can't run `npm start -- --inspect` command for debug, because it is provided only 1 argument

Comment (giorgiomazzei79) [reply scripts/develop.sh]: ok
